### PR TITLE
Issues/8/replicate k8s cicd

### PR DESCRIPTION
**Pull request by yahya130**
Creating helm charts and refining pipeline

### CODE REVIEW

**Review by keijoraamat — APPROVED** [review]

File: .github/workflows/reusable-helm-package-upload.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Helm
+        uses: azure/setup-helm@v1
+
+      - name: Lint Helm chart
+        run:  helm lint $helm_chart_directory_path/${{ inputs.app_name }}
+
+      #ToDo: sign chart
```
Comment (keijoraamat) [inline .github/workflows/reusable-helm-package-upload.yml:30]:
```suggestion
      #ToDo: sign chart
```

Comment (keijoraamat) [inline .github/workflows/reusable-helm-package-upload.yml:30]:
```suggestion
      #ToDo: sign chart
      #ToDo: versioning system
```

File: helm-charts/dmr/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```
Comment (guypritchard) [inline helm-charts/dmr/templates/NOTES.txt:1]: I love these files (and don't change this now) - but could these be markdown?  Notes for future humans.

**Review by guypritchard — APPROVED** [review]
This is difficult to review - but looks okay to me.  Perhaps needs final sign off from @decodingahmed to determine this is 100% awesome.

File: helm-charts/dmr/templates/NOTES.txt (hunk 2 above)
Comment (yahya130) [reply helm-charts/dmr/templates/NOTES.txt:1]: Sure thing, just a little reformatting needed, there is an item here https://github.com/buerokratt/Infrastructure/issues/24 for further work on the helm charts I could capture this here :)

**Pull request merged**